extract_unique_sensors_01.py

```diff
@@ -62,17 +62,28 @@ def extract_all_sensors(sensors_data, sensor_name):
     return candidate_lines
 
 # Loop over candidate_lines and extract sensor names
-def splitting_list(candidate_sensors, sensor_name):
-    """Splits the candidate_lines by dot and append that to sensors_list. It also removes the 
+def splint_extracted_sensors(candidate_sensors, sensor_name):
```

Comment:
Not sure I understand, but what's splint?
Attention to detail, please review your code before pushing.

Do:
`git diff`

Before `git push`.

Reply:
It's a typing error, I will fix it. Thanks